doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -454,933 +536,640 @@ includes considerations specific to deployments at the edge of the network. The
+     -
+   * - `7.9.7`
+     - sec.mon.015
+     - The Platform **must** ensure that the Monitoring systems are never starved of resources and **must** activate alarms when resource utilisation exceeds a configurable threshold.
+     -
+   * - `7.9.7`
+     - sec.mon.016
+     - The Platform Monitoring components **should** follow security best practices for auditing, including secure logging and tracing.
+     -
+   * - `7.9.7`
+     - sec.mon.017
+     - The Platform **must** audit systems for any missing security patches and take appropriate actions.
+     - `5.3.3 Vulnerability assessment`
+   * - `7.9.7`
+     - sec.mon.018
+     - The Platform, starting from initialization, **must** collect and analyze logs to identify security events, and store these events in an external system.
+     - `5.3.4 Patch management`
+   * - `7.9.7`
+     - sec.mon.019
+     - The Platform's components **must not** include an authentication credential, e.g., password, in any logs, even if encrypted.
+     -
+   * - `7.9.7`
+     - sec.mon.020
+     - The Platform's logging system **must** support the storage of security audit logs for a configurable period of time.
+     -
+   * - `7.9.7`
+     - sec.mon.021
+     - The Platform **must** store security events locally if the external logging system is unavailable and shall periodically attempt to send these to the external logging system until successful.
+     -
+   * - `7.9.8`
+     - sec.oss.001
+     - Open source code **must** be inspected by tools with various capabilities for static and dynamic code analysis.
+     - `5.3.3 Vulnerability assessment`
+   * - `7.9.8`
+     - sec.oss.002
+     - The `CVE (Common Vulnerabilities and Exposures) <https://cve.mitre.org/>`__ **must** be used to identify vulnerabilities and their severity rating for open source code part of Cloud Infrastructure and workloads software.
+     -
+   * - `7.9.8`
+     - sec.oss.003
+     - Critical and high severity rated vulnerabilities **must** be fixed in a timely manner. Refer to the `CVSS (Common Vulnerability Scoring System <https://www.first.org/cvss/>`__ to know a vulnerability score and its associated rate (low, medium, high, or critical).
+     -
+   * - `7.9.8`
+     - sec.oss.004
+     - A dedicated internal isolated repository separated from the production environment **must** be used to store vetted open source content.
+     - `5.13 Trusted Registry`
+   * - `7.9.8`
+     - sec.oss.005
+     - A Software Bill of Materials (`SBOM <https://www.ntia.gov/SBOM>`__) **should** be provided or build, and maintained to identify the software components and their origins.
+     -
+   * - `7.9.9`
+     - sec.arch.001
+     - Threat Modelling methodologies and tools **should** be used during the Secure Design and Architecture stage triggered by Software Feature Design trigger. It may be done manually or using tools like open source OWASP Threat Dragon
```

Comment:
```suggestion
     - Threat Modelling methodologies and tools **should** be used during the Secure Design and Architecture
        stage triggered by Software Feature Design trigger. It may be done manually or using tools like open source
        OWASP Threat Dragon
```